test/TableGatewayAbstractFactoryTest.php

```diff
@@ -149,7 +150,12 @@ public function validConfig()
      */
     public function testFactoryReturnsTableGatewayInstanceBasedOnConfiguration($adapterServiceName)
     {
-        $hydrator = $this->prophesize(ClassMethods::class)->reveal();
+        if (class_exists(ClassMethodsHydrator::class)) {
```

Comment:
It might be better to extract it to separate method, like: `getHydrator()`  (private) so the logic is just in one place.

Reply:
I've added `getClassMethodsHydratorClassName()`, as similar logic was also needed for an assertion